descriptors/ModuleDescriptor-template.json

```diff
@@ -122,6 +122,22 @@
         }
       ]
     },
+    {
+      "id": "resolve-claim-as-missing",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": ["POST"],
+          "pathPattern": "/circulation/loans/{id}/resolve-claim-as-missing",
+          "permissionsRequired": [
+            "circulation.loans.resolve-claim-as-missing.post"
+          ],
+          "modulePermissions": [
+            "modperms.circulation.loans.resolve-claim-as-missing.post"
+          ]
+        }
+      ]
+    },
```

Comment:
Given that this endpoint is only relevant as part of the claimed returned process, why is it defined a separate interface rather than as part of `claim-item-returned`?

Reply:
Moved to the claimed-returned API definition file.